batch/data-plane/.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: java
```

Comment:
Why do we need this travis.yml file?

Reply:
I noted this also. My thinking was that we would clean up docs, remove redundant files, etc after we get the repo imported.

Reply:
I'm fine with follow-up clean-up.